src/components/UI/ScrollArea.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { describe, it } from "vitest";
```

Comment:
```suggestion

```

Reply:
These imports are handled automatically by defining `globals: true` in the Vitetest config.

To get TypeScript to recognize these types, add the following line to `/tsconfig.json`:

```
{ 
  "compilerOptions": {
    ...,
    "types": ["vitest/globals"]
  }
}
```